packages/yoga/src/Dialog/web/Header.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { Card } from '../..';
+
+const Header = props => (
+  <Card.Header
+    ta="center"
+    color="text.primary"
+    fw="medium"
+    fs="xlarge"
+    mb="large"
+    {...props}
+  />
```

Comment:
I'm not sure if in this case it worth to use Card's Header (if we update Card's one this will break), maybe we could bring this to PD team in order to define maybe a standard for these headers? Because if it happens we can then have a separate Header which both Card and Dialog uses.